src/Symfony/Component/VarExporter/Tests/Fixtures/LazyProxy/Php82NullStandaloneReturnType.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Symfony\Component\VarExporter\Tests\Fixtures\LazyProxy;
```

Comment:
missing file header docblock

Reply:
I was thinking this was enforced by csfixer?
they are missing in a lot of fixture files.

whatever, I added it in the two files I created :+1: